src/mudlet-lua/lua/geyser/GeyserUserWindow.lua

```diff
@@ -103,6 +113,10 @@ function Geyser.UserWindow:new(cons)
       me:resize(cons.width,cons.height)
   end
 
+  if me.titleText then
```

Comment:
If you change the constructor and remove the titleText, the user window's title won't be reset, I think.